zipline/pipeline/common.py

```diff
@@ -1,6 +1,7 @@
 """
 Common constants for Pipeline.
 """
+ACTUAL_VALUE_FIELD_NAME = 'actual_value'
```

Comment:
I don't think we really agreed on a specific name - seemed like a logical choice to me, since the other columns (mean, low, high, stdev) are estimates of this value, which is the "actual" value.